qutebrowser/completion/completionwidget.py

```diff
@@ -192,10 +192,13 @@ def _next_prev_item(self, prev):
         Args:
             prev: True for prev item, False for next one.
         """
-        idx = self._next_idx(prev)
-        qtutils.ensure_valid(idx)
```

Comment:
I think we don't have a test for the removal of this yet, i.e the crash when doing `:foo<tab>` with `auto-open` and `show` being true?